src/Raven.Client/Documents/Session/Loaders/IncludeBuilder.cs

```diff
@@ -9,16 +9,42 @@
 
 namespace Raven.Client.Documents.Session.Loaders
 {
+    /// <summary>
+    /// The server is instructed to pre-load referenced documents concurrently with retrieving the documents.<br/>
+    /// The documents are added to the session unit of work, and subsequent requests to load them are served directly from the session cache,
+    /// without requiring any additional queries to the server. <br />
+    /// The server can then be instructed to pre-load the referenced object at the same time that the root object is retrieved, for example using:
+    /// <example>
+    /// <code>
+    /// Order order = session.Include&lt;Order&gt;(x => x.CustomerId).Load("orders/1-A");
+    /// // this will not require querying the server:
+    /// Customer customer = session.Load&lt;Customer&gt;(order.CustomerId);
+    /// </code></example>
+    /// </summary>
+    /// <inheritdoc cref="DocumentationUrls.Session.Querying.Includes"/>
```

Comment:
Does _everything_ will inherit from this comment now? Even counter and time series includes? Also includes requested for queries?

Reply:
Hmm, most of the includes are only for documents, I see I set inherit on some includes with builder. I will change them to be more generic. (like with list of possible includes etc).

Reply:
I only did the document parts, didn't touch the timeseries/counters/revisions. They have own methods inside builder and are not documented yet (and not overwritten by generic descripton). However, includes (without builder in parameter) are for documents and the description is valid for query/load.